dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/AccessTokenController.java

```diff
@@ -154,10 +155,10 @@ public Result queryAccessTokenList(@Parameter(hidden = true) @RequestAttribute(v
     @GetMapping(value = "/user/{userId}")
     @ResponseStatus(HttpStatus.OK)
     @ApiException(QUERY_ACCESSTOKEN_BY_USER_ERROR)
-    public Result queryAccessTokenByUser(@Parameter(hidden = true) @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
-                                         @PathVariable("userId") Integer userId) {
-        Map<String, Object> result = this.accessTokenService.queryAccessTokenByUser(loginUser, userId);
-        return this.returnDataList(result);
+    public Result<List<AccessToken>> queryAccessTokenByUser(@Parameter(hidden = true) @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
+                                                            @PathVariable("userId") Integer userId) {
+        List<AccessToken> accessTokens = accessTokenService.queryAccessTokenByUser(loginUser, userId);
+        return Result.success(accessTokens);
```

Comment:
If we do this refactor and only throw exception when meet some unexpected operation, UI will not display alert box with error message, and I think it's not user-friendly.

Reply:
We have `ApiExceptionHandler` to transform the ServiceException to standard Result.